CIPs/cip-remove_tobin_tax.md

```diff
@@ -0,0 +1,71 @@
+---
+cip: <to be assigned>
+title: Remove Tobin Tax
+author: Paul Lange (@palango)
+discussions-to: <URL>
+status: Draft
+type: Standards Track
+category: Ring 0
+created: 2023-03-03
+license: Apache 2.0
+---
+
+## Simple Summary
+
+Remove the [Tobin Tax](https://docs.celo.org/protocol/stability/tobin-tax) from the Celo protocol. This removes a source of potential confusion for users and simplifies the protocol.
+
+## Abstract
+
+This CIP proposes to remove the Tobin Tax from the Celo protocol. The Tobin Tax is a small fee that is levied on CELO transfers when the reserve ratio falls below a certain value. The Tobin tax has never been enabled on mainnet.
+
+Currently the client includes a special transfer function that checks if a Tobin Tax has to be paid and pays it if necessary. This code is proposed to be removed.
+
+## Motivation
+
+The Tobin Tax creates two problems.
+
+First, it can lead to confusion for users, when two equal transactions result in different outcome, depending on whether or not the Tobin Tax has to be paid.
```

Comment:
```suggestion
First, it can lead to confusion for users, when two equal transactions result in a different outcome, depending on whether or not the Tobin Tax has to be paid.
```